dataflow/src/main/java/org/checkerframework/dataflow/analysis/Analysis.java

```diff
@@ -743,24 +756,33 @@ public boolean isRunning() {
         return nodeValues.get(n);
     }
 
-    /** Return all current node values. */
-    public IdentityHashMap<Node, A> getNodeValues() {
+    /**
+     * Return all current node values.
+     *
+     * @return {@link #nodeValues}
+     */
+    public IdentityHashMap<Node, V> getNodeValues() {
         return nodeValues;
     }
 
-    /** Set all current node values to the given map. */
-    /*package-private*/ void setNodeValues(IdentityHashMap<Node, A> in) {
+    /**
+     * Set all current node values to the given map.
+     *
+     * @param in the current node values
+     */
+    /*package-private*/ void setNodeValues(IdentityHashMap<Node, V> in) {
         assert !isRunning;
         nodeValues.clear();
         nodeValues.putAll(in);
     }
 
     /**
+     * @param t a {@link Tree} t
```

Comment:
Links to the types are unnecessary in the description - javadoc already links the types in the signature.